tests/ldevnet.go

```diff
@@ -2,34 +2,51 @@ package tests
 
 import (
 	"context"
+	"fmt"
 	"testing"
+	"time"
 
 	"github.com/filecoin-project/go-address"
-	"github.com/filecoin-project/lotus/build"
 	"github.com/filecoin-project/lotus/chain/types"
-	"github.com/textileio/powergate/ldevnet"
+	"github.com/ory/dockertest"
+	"github.com/textileio/lotus-client/api/apistruct"
+	"github.com/textileio/powergate/lotus"
+	"github.com/textileio/powergate/util"
 )
 
-func init() {
-	build.InsecurePoStValidation = true
-}
-
-func CreateLocalDevnet(t *testing.T, numMiners int) (*ldevnet.LocalDevnet, address.Address, []address.Address, func()) {
-	dnet, err := ldevnet.New(numMiners, ldevnet.DefaultDuration)
+func CreateLocalDevnet(t *testing.T, numMiners int) (*apistruct.FullNodeStruct, address.Address, []address.Address) {
```

Comment:
So now `CreateLocalDevnet` spinup a docker container instead of running the devnet per-se.
The good part is that all code called this method, so changing this was indistinguishable for _the clients_.